NAMESPACE

```diff
@@ -6,11 +6,11 @@ export(add_layer)
 export(add_root)
 export(hierplane)
 export(hierplaneOutput)
-export(hierplane_settings)
 export(hierplane_styles)
 export(hp_dataframe)
 export(hp_datatree)
 export(hp_spacyr)
 export(renderHierplane)
+export(spacyr_default_attributes)
```

Comment:
I think the pkgdown yaml file needs to be updated to included this new function

Reply:
Done!